src/patternfly/components/ActionList/action-list.scss

```diff
@@ -0,0 +1,42 @@
+.pf-c-action-list {
+  --pf-c-action-list--spacer: var(--pf-global--spacer--md);
+  --pf-c-action-list__group-group--spacer: calc(var(--pf-global--spacer--2xl) - var(--pf-global--spacer--md));
+  --pf-c-action-list__group-group--m-vertical--Gap: var(--pf-global--spacer--lg);
+  --pf-c-action-list--m-vertical--spacer: var(--pf-global--spacer--sm);
+
+  &.pf-m-icons {
+    --pf-c-action-list--spacer: 0;
+  }
+}
+
+.pf-c-action-list,
+.pf-c-action-list__group {
+  display: flex;
+
+  > *:not(:last-child) {
```

Comment:
Can you nest an action-list in another action list? Seems like you wouldn't, you would just nest groups?

Reply:
yep you would only nest groups, moving this outside of this double selector

Reply:
yep just groups, do you suggest changing this